src/dataSources/JSON.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var Null = require('./Null');
```

Comment:
@joneit Actually I can't recall our discussion here. Do you still want me to remove it?